ecommerce/extensions/catalogue/migrations/0050_bin_list_attribute.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.16 on 2021-09-10 16:54
```

Comment:
We can actually delete this migration and add the attribute directly to the product type via django-oscar.